src/tidysic.py

```diff
@@ -59,7 +79,7 @@ def parse_in_directory(dir_src, with_album):
             else:
                 artists[artist][title] = f
         else:
-            print(f'Could not get artist: {title}')
+            print(f'Could not move the file: {f}')
```

Comment:
Is this right ?

Reply:
Nope that was not the right information to display ; fixed!